bionic-arm64/Dockerfile

```diff
@@ -25,32 +30,12 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   libsecret-1-dev:arm64 \
   curl \
   gnupg \
-  unzip \
-  # Dependencies for building python from source
-  build-essential \
-  libreadline-gplv2-dev \
-  libncursesw5-dev \
-  libssl-dev \
-  libsqlite3-dev \
-  tk-dev \
-  libgbm-dev \
-  libc6-dev \
-  libbz2-dev \
-  libffi-dev \
-  zlib1g-dev
+  unzip
 
-# Compile python3.9 from source
-RUN curl -O https://www.python.org/ftp/python/3.9.4/Python-3.9.4.tar.xz
-RUN tar -xf Python-3.9.4.tar.xz
-RUN cd Python-3.9.4 && ./configure --enable-optimizations && make altinstall
-RUN update-alternatives --install /usr/bin/python python /usr/local/bin/python3.9 1
+# Set python3 as default
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
```

Comment:
the default python3 version in bionic is 3.6.9
is that new enough for your uses?